tests/foreman/remote_DB/test_remote_DB.py

```diff
@@ -0,0 +1,228 @@
+"""Test class for Remote DB feature
+
+:Requirement: Offload DBs from Satellite Server
```

Comment:
Nitpick : we generally keep it single work.

Reply:
I just use one which was already used by adujicek in polarion. And it's not true that we keep it one word. Lot of test have 2+ words in requirements(I remember checking it :)).

Reply:
Thanks. I mean to say shorter word :) It's fine then if we have it already in polarion.

Reply:
@lvrtelov is right, there is no guideline to keep Requirement at one word. Our requirements in general are kind of messy, there is no consistency in them.
@sghai asked on satqe list to re-use requirements wherever possible, so using existing one here is definitely right choice.